File: backend/services/grading-service/src/main/java/com/lmspilot/grading/GradingServiceApplication.java
package com.lmspilot.grading;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class GradingServiceApplication {
    public static void main(String[] args) { SpringApplication.run(GradingServiceApplication.class, args); }
}

File: backend/services/grading-service/src/main/kotlin/com/lmspilot/grading/GradingServiceApplication.kt
package com.lmspilot.grading

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class GradingServiceApplication

fun main(args: Array<String>) {
    runApplication<GradingServiceApplication>(*args)
}
